test_logger_msg.py

```diff
@@ -0,0 +1,39 @@
+from lb_logger import logger
+import unittest
+from labelbox import Client
+import logging
+import os
+
+api_key = os.environ['apikey']
+
+class TestLoggerMessages(unittest.TestCase):
+    "Test that the logger's out put will match the expected levels set by logger"
+
+    def setUp(self):
+        self.msg = "Test message"
+        self.test_logger = logger
+    
+    def test_info_msg(self) -> None: 
+        with self.assertLogs(self.test_logger) as assert_log:
+            self.test_logger.info(self.msg)
+        self.assertEqual(assert_log.output, ['INFO:lb_logger:%s' %(self.msg)])
+
+    def test_warning_msg(self) -> None: 
+        with self.assertLogs(self.test_logger) as assert_log:
+            self.test_logger.warning(self.msg)
+        self.assertEqual(assert_log.output, ['WARNING:lb_logger:%s' %(self.msg)])    
+
+    def test_error_msg(self) -> None: 
+        with self.assertLogs(self.test_logger) as assert_log:
+            self.test_logger.error(self.msg)
+        self.assertEqual(assert_log.output, ['ERROR:lb_logger:%s' %(self.msg)])    
+
+    def test_critical_msg(self) -> None:      
+        with self.assertLogs(self.test_logger) as assert_log:
+            self.test_logger.critical(self.msg)
+        self.assertEqual(assert_log.output, ['CRITICAL:lb_logger:%s' %(self.msg)])     
+
+    def test_verbosity(self) -> None:     
```

Comment:
i am a bit unsure on how to confirm this test. i had to insert my own `api_key` variable here to confirm the test. is there a good way to ensure that a call to Client does not need an actual api key when running with pytest?